### PR TITLE
Update of source parameter in documentation

### LINKED ISSUE #1237: [DOC] `_source` input parameter incorrect/unclear
**What do you want to do?**
 
- [ ] Request a change to existing documentation
- [ ] Add new documentation
- [X] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.

OpenSearch 2.2.1

1.  On many of the document API pages, such as <https://opensearch.org/docs/latest/opensearch/rest-api/document-apis/update-document/>, it says `_source` is an input parameter which defines "whether to include the _source field in the response body. Default is true." However it says it is of type List, not Boolean.

    Looking at Elastic docs, they say that this parameter can also be used like `_source_includes` instead of as a Boolean. This is presumably why it's List-typed, but OpenSearch docs do not mention this afaict. It seems like it should say "List or Boolean".

1.  The "Update document" page says that `_source` defaults to true. This seems to be incorrect; it looks like it defaults to false.

    ```
    $ curl -X POST -H 'Content-Type: application/json' -u admin:admin http://localhost:9200/ns_test_indexes_test-index/_update/7ZiiQIMB88D4nmsFfzEJ -d '{"doc":{"hello":"world3"}}'
    {"_index":"ns_test_indexes_test-index","_id":"7ZiiQIMB88D4nmsFfzEJ","_version":4,"result":"updated","_shards":{"total":2,"successful":1,"failed":0},"_seq_no":3,"_primary_term":1}⏎
    $ curl -X POST -H 'Content-Type: application/json' -u admin:admin http://localhost:9200/ns_test_indexes_test-index/_update/7ZiiQIMB88D4nmsFfzEJ -d '{"doc":{"hello":"world3"},"_source":true}'
    {"_index":"ns_test_indexes_test-index","_id":"7ZiiQIMB88D4nmsFfzEJ","_version":4,"result":"noop","_shards":{"total":0,"successful":0,"failed":0},"_seq_no":3,"_primary_term":1,"get":{"_seq_no":3,"_primary_term":1,"found":true,"_source":{"jshfsdhfkdata":{"hello":"world"},"hello":"world3"}}}⏎
    ```

Thank you 🙇‍♂️

### PR DESCRIPTION
**Pull request by ariamarble**
Signed-off-by: ariamarble <armarble@amazon.com>

### Description

Updates source parameter in documentation.

### Issues Resolved

#1237 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _opensearch/rest-api/document-apis/update-document.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ require_alias | Boolean | Specifies whether the destination must be an index ali
 refresh | Enum | If true, OpenSearch refreshes shards to make the operation visible to searching. Valid options are `true`, `false`, and `wait_for`, which tells OpenSearch to wait for a refresh before executing the operation. Default is `false`. | No
 retry_on_conflict | Integer | The amount of times OpenSearch should retry the operation if there's a document conflict. Default is 0. | No
 routing | String | Value to route the update operation to a specific shard. | No
-_source | List | Whether to include the `_source` field in the response body. Default is true. | No
+_source | Boolean or List | Whether to include the `_source` field in the response body. Default is `false`. This parameter can also be used with a comma-separated list of source fields to include in the query response. | No
```
Comment (alicejw-aws) [inline _opensearch/rest-api/document-apis/update-document.md:57]: nit. . .might just say "whether or not"

**Review by alicejw-aws — APPROVED** [review]
Looks good!

File: _opensearch/rest-api/document-apis/update-document.md (hunk 1 above)
Comment (cwillum) [inline _opensearch/rest-api/document-apis/update-document.md:57]: Just suggestions.
"Specifies whether to populate the `_source` field in the response body."? 
Maybe, "This parameter also supports a comma-separated list of source fields for including multiple source fields in the query response."

**Review by cwillum — APPROVED** [review]
Just a few suggestions. But looks good.

**Pull request merged**